engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -42,9 +42,10 @@ To install Docker EE, you need the 64-bit version of one of these Ubuntu version
 - Xenial 16.04 (LTS)
 - Trusty 14.04 (LTS)
 
-Docker EE is supported on `x86_64` (or `amd64`) and `s390x` (IBM Z) architectures.
+Docker EE is supported on `x86_64` (or `amd64`), `s390x` (IBM Z), and `ppc64el`
+(IBM POWER) architectures.
 
-The only supported storage driver for Docker EE on Ubuntu is `aufs`.
+Docker EE on Ubuntu supports `overlay2` and `aufs` storage drivers.
```

Comment:
Should we be more explicit about `aufs`? i.e.; for kernel 4 and up, `overlay2` is the preferred driver. For kernel 3.13, `aufs` is supported (because it doesn't support overlay/overlay2)

Reply:
Done.